UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/helper/MetricHelper.js

```diff
@@ -0,0 +1,100 @@
+import { Gauge, Trend, Counter } from 'k6/metrics';
+import { group } from 'k6';
+
+const s63SmallEssCreationTime = new Trend('SmallEssCreationTime');
+const s63SmallEssBatchTime = new Trend('SmallEssBatchTime');
+const s63SmallEssDownloadTime = new Trend('SmallEssDownloadTime');
+const s63SmallEssProductCount = new Gauge('SmallEssProductCount');
+const s63SmallEssSizeRange = new Gauge('SmallEssSizeRange');
+const s63SmallEssE2E = new Trend('SmallEssE2ETime_Minutes');
+
+const s63MediumEssCreationTime = new Trend('MediumEssCreationTime');
+const s63MediumEssBatchTime = new Trend('MediumEssBatchTime');
+const s63MediumEssDownloadTime = new Trend('MediumEssDownloadTime');
+const s63MediumEssProductCount = new Gauge('MediumEssProductCount');
+const s63MediumEssSizeRange = new Gauge('MediumEssSizeRange');
+const s63MediumEssE2E = new Trend('MediumEssE2ETime_Minutes');
+
+const s63LargeEssCreationTime = new Trend('LargeEssCreationTime');
+const s63LargeEssBatchTime = new Trend('LargeEssBatchTime');
+const s63LargeEssDownloadTime = new Trend('LargeEssDownloadTime');
+const s63LargeEssProductCount = new Gauge('LargeESSProductsCount');
+const s63LargeEssSizeRange = new Gauge('LargeEssSizeRange');
+const s63LargeEssE2E = new Trend('LargeEssE2ETime_Minutes');
+
+const productIdentifiersTime = new Trend('ESCreation_PI');
+const productVersionsTime = new Trend('ESCreation_PV');
+const sinceDateTime = new Trend('ESCreation_SD');
+const newfilespublishedTime = new Trend('ESCreation_Publish');
+const healthTime = new Trend('Health');
+
+const productIdentifiersCounter = new Counter('Counts_PI');
+const productVersionsCounter = new Counter('Counts_PV');
+const sinceDateTimeCounter = new Counter('Counts_SD');
+const newfilespublishedCounter = new Counter('Counts_Publish');
+const healthCounter = new Counter('Counts_Health');
+
+export function SmallESSMetrics(prodCount, fileSizeInMB, ESSReponseTime, FSSBatchResponseTime, FSSDownloadTime) {
```

Comment:
For all the functions "FSSDownloadTime" is unused.

Reply:
Removed, not required as part of ESS PT